template.yml

```diff
@@ -0,0 +1 @@
+{"awsTokens":"sfvdvkje43"}
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Code Generic   
**Description:** Code Generic Credential   
**Severity:** HIGH   
**Match:** {"awsTokens":"**********"}  
[This comment was created by Aqua Pipeline]